src/compute/src/sink/persist_sink.rs

```diff
@@ -1002,7 +1021,8 @@ where
             incomplete: Option<BatchBuilder<SourceData, (), Timestamp, Diff>>,
         }
 
-        let mut in_flight_batches = HashMap::<
+        #[allow(clippy::disallowed_types)]
```

Comment:
Can you add a comment spelling out that/why we are not worried about non-deterministic iteration here?

Reply:
will do!